tests/src/Kernel/Update405Test.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\Tests\lightning\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\lightning\Update\Update405;
+
+/**
+ * @group lightning
+ */
+class Update405Test extends KernelTestBase
```

Comment:
I think we should add a `@coversDefaultClass \Drupal\lightning\Update\Update405` to the doc comment of this class.